src/components/MetadataCollections/MetadataCollections.js

```diff
@@ -253,9 +306,10 @@ class MetadataCollections extends React.Component {
                         </div>
                         <Button
                           buttonStyle="none"
-                          disabled={disableReset()}
+                          // disabled={disableReset()}
                           id="clickable-reset-all"
-                          onClick={resetAll}
+                          // onClick={resetAll}
+                          onClick={() => this.resetAll(getFilterHandlers(), getSearchHandlers(), resetAll)}
```

Comment:
Here this.resetAll() is called with three parameters, but its specification only has two.

Reply:
the unused parameter has been deleted

Reply:
You forgot remove the parameter here in MetadataCollections.js (you did remove it just in MetadataSources.js).